src/arguments/arguments.go

```diff
@@ -19,7 +19,7 @@ type ArgumentList struct {
 	CABundleFile      string `default:"" help:"Alternative Certificate Authority bundle file"`
 	CABundleDir       string `default:"" help:"Alternative Certificate Authority bundle directory"`
 	PartitionFilter   string `default:"[\"Common\"]" help:"JSON array of partitions to collect"`
-	AuthHost          string `default:"" help:"Hostname or IP address of the Big IQ auth token provider."`
+	AuthHost          string `default:"localhost" help:"Hostname or IP address of the Big IQ auth token provider."`
```

Comment:
I think having this default to `localhost` is a breaking change. Most likely, we'd want this to default to whatever `Hostname` is set to.